base/docs/helpdb/Base.jl

```diff
@@ -1163,7 +1163,7 @@ randsubseq!
 """
     maximum(A, dims)
 
-Compute the maximum value of an array over the given dimensions.
+Compute the maximum value of an array over the given dimensions. Also see :func:`max`.
```

Comment:
This needs to be in Markdown link syntax, and should probably give some explanation
```
See also the [`max(a,b)`][:func:`max`] function to take the maximum of
two or more arguments, which can be applied elementwise to arrays via `max.(a,b)`.
```